client/src/pages/InsightsPage/InsightsPage.js

```diff
@@ -78,6 +79,7 @@ const InsightsPage = () => {
 
   return (
     <div>
+      <PageHeader pageTitle={"Insights"} />
```

Comment:
No change is needed here but something to consider in the future is to perhaps have a general page template where it takes in a prop with the pageTitle so then you won't need to add PageHeader all the time. But for our case, since it's just 3 pages that require the PageHeader, it's okay to have it here.